src/app/packages/overlays/effects/overlays.effects.ts

```diff
@@ -78,7 +80,16 @@ export class OverlaysEffects {
 		.ofType<LoadOverlaysAction>(OverlaysActionTypes.LOAD_OVERLAYS)
 		.switchMap((action) => {
 			return this.overlaysService.search(action.payload)
-				.map(data => new LoadOverlaysSuccessAction(data))
+				.mergeMap( (overlays: OverlaysFetchData) => {
```

Comment:
![image](https://user-images.githubusercontent.com/5757359/34194645-60c88ffc-e562-11e7-92bd-7d4655a1d9d9.png)

Reply:
![image](https://user-images.githubusercontent.com/11487010/34201204-66cf1e10-e57c-11e7-83fc-12fe5c350f2a.png)

Reply:
Will be fixed... ahm...